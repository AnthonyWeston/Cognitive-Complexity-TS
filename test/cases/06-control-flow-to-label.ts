function h() {
    label1:
    while (true) {
        break label1;
    }

}

function i() {
    label2:
    while (true) {
        continue label2;
    }
}
