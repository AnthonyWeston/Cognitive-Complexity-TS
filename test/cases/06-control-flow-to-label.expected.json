{
    "06-control-flow-to-label.ts": {
        "score": 4,
        "inner": [
            {
                "name": "h",
                "score": 2,
                "line": 1,
                "column": 1
            },
            {
                "name": "i",
                "score": 2,
                "line": 9,
                "column": 1
            }
        ]
    }
}